CHANGELOG.md

```diff
@@ -15,6 +15,26 @@ FEATURES:
 * Kubernetes 1.24
   * Add support for Kubernetes 1.24 where ServiceAccounts no longer have long term JWT tokens. [[GH-1431](https://github.com/hashicorp/consul-k8s/pull/1431)]
   * Upgrade kubeVersion in helm chart to support Kubernetes 1.21+.
+  NOTE: Users deploying multiple services to the same Pod (multiport) must also deploy an Kube Secret for each ServiceAccount associated with the Consul service. The name of the Secret must match the ServiceAccount name and be of type 'kubernetes.io/service-account-token'
+Example:
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: svc1
+  annotations:
+    kubernetes.io/service-account.name: svc1
+type: kubernetes.io/service-account-token
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: .multiport
```

Comment:
nit: secret names can't have `.`'s can they?

Reply:
Oof good catch this was a poor copy paste lol